vendor/visualmetrics.py

```diff
@@ -1521,11 +1544,18 @@ def calculate_perceptual_speed_index(progress, directory):
         ssim = compute_ssim(current_frame, target_frame)
         gc.collect()
         last_ms = p['time']
-    return int(per_si)
+        completeness_value.append((p['time'], int(per_si)))
+
+    per_si = int(per_si)
+    raw_progress_value = ["0=0"]
+    for timestamp, percent in completeness_value:
+        p = int(100 * float(percent) / float(per_si))
+        raw_progress_value.append('%d=%d' % (timestamp, p))
+
+    return per_si, ", ".join(raw_progress_value)
 
 
 def calculate_hero_time(progress, directory, hero, viewport):
-  try:
```

Comment:
The try/except needs to be included in the PR. We wrap the hero times because it fails on Arabic Wikipedia making all visual metrics fail. Think try/except is needed both hereto and Contentful speed index

Reply:
Ok, I've added this back. The contentful speed index function has it as well.